app/controlls/List.js

```diff
@@ -152,9 +152,32 @@ SDL.List = Em.ContainerView.extend({
                 classNames: 'list-item',
 
                 classNameBindings: [
-                  'this.voiceOver'
+                  'this.voiceOver',
+                  'dayMode',
+                  'nightMode',
+                  'highLightedMode'
                 ],
 
+                dayMode:false,
+                nightMode:false,
+                highLightedMode:false,
+
+                setMode:function(mode) {
+                  this.set('dayMode',false);

```

Comment:
@YarikMamykin can be simplified
```suggestion
   mode = SDL.SDLModel.data.imageModeList.includes(mode) ? mode : SDL.SDLModel.data.imageModeList[0];
   this.set('dayMode', mode ==  SDL.SDLModel.data.imageModeList[0]);
   this.set('nightMode', mode ==  SDL.SDLModel.data.imageModeList[1]);
   this.set('highLightedMode', mode ==  SDL.SDLModel.data.imageModeList[2]);
     
   }

```

Reply:
@AKalinich-Luxoft applied in https://github.com/LuxoftSDL/sdl_hmi/pull/10/commits/c5e3d3d74828dc39089d4ba7e2ec41e10573a8e0